config/routes.rb

```diff
@@ -3,5 +3,10 @@
   root 'welcome#index'
 
   get '/auth/:provider/callback', to: 'sessions#create'
+  namespace :user do
+    resources :dashboard, only: :show
+  end
   get '/dashboard', to: 'dashboard#show'
+
+  resources :households, only: %i[new create]
```

Comment:
Will the household dashboard be the household show page?  Or is there where '/dashboard' redirects to, I forget?

Reply:
@denverdevelopments  yes, the household show page and the household dashboard page are the same! They should use the `/dashboard` route